x-pack/packages/ml/landing_links/src/landing_links.test.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { render } from '@testing-library/react';
+import { SecurityPageName, mockNavigateTo, mockGetAppUrl } from '@kbn/security-solution-navigation';
+import { LandingColumnLinks } from './landing_links';
+import type { NavigationLink } from '@kbn/security-solution-navigation';
+
+jest.mock('../navigation');
```

Comment:
`navigation` is in the security-solution-navigation package and I am not sure how to properly use it here. Any recommendations?